src/helpers/FormatTemperature.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+
+import formatTemperature from '../../helpers/FormatTemperature';
+
+describe('Formats Temperature Number Correctly', () => {
+    
+    expect(formatTemperature("dave")).toBe("N/A");
+
+  });
```

Comment:
Unit tests would ideally test all of the cases that we care about, so in this case it would be something like;

```ts
describe('Format Temperature', () => {
    it('should round a number and convert to string with unit', () => {
        // test code goes here
    }); 

    it('should return N/A if the input is null', () => {
        // test code goes here
    });
});
```